$NetBSD: patch-av,v 1.3 2009/05/13 21:00:25 markd Exp $

GridEngine dev mailing list patch for "nbsd-1" issue, along with aimk change

--- source/libs/uti/sge_arch.c.orig	2009-04-09 13:19:28.000000000 +1200
+++ source/libs/uti/sge_arch.c	2009-04-09 10:10:44.000000000 +1200
@@ -72,7 +72,7 @@
 #define xstr(s) str(s)
 #define str(s) #s
 
-   return xstr(SGE_ARCH_STRING);
+   return (SGE_ARCH_STRING);
 }
 
 /****** uti/prog/sge_get_root_dir() *******************************************
